app/src/main/java/com/dessalines/thumbkey/utils/Types.kt

```diff
@@ -96,6 +96,7 @@ enum class KeyboardLayout(val title: String, val index: Int) {
     ThumbKeyENv4("Thumb-Key english v4", 0),
     ThumbKeyENv4Symbols("Thumb-Key english v4 with symbols", 21),
     ThumbKeyENv4Programmer("Thumb-Key english v4 (programmer)", 1),
+    ThumbKeyENv4Multi("Thumb-Key english v4 (EN+SK multi)", 48),
```

Comment:
Add this at the bottom of the list please, the front end sorts alphabetically anyway.

Reply:
okay, done.